templates/ICON.jinja

```diff
@@ -11,6 +11,7 @@
             "member_type": "{{member_type}}",
             "factor": 5,
             "file_specification": [{
+                "GRIB": {"format": "GRIB", "time_dim": "step", "horizontal_dims": ["values"], "var_excl": ["tlon", "tlat", "vlon", "vlat", "ulon", "ulat", "h", "slor", "anor", "isor", "sdor"], "fill_value_key": "_FillValue"},
```

Comment:
I think at some point we may want to come up with a more flexible solution for `var_excl`, so that we don't have to hardcode it here. But I think this is not in the scope of this PR :+1:

Reply:
I am open for suggestions. Maybe worth to raise an issue?

Reply:
I cannot remember now what the actual error was if those variables were not excluded, but at the time we were pretty sure that we did not need to test those values, since they seemed to be dimensions/coordinates. If I remember correctly from what Andrea told me, in GRIB there is no distinction between variables and dimensions/coordinates as in NetCDF, do you know if this is true?

I think that the names of these variables are probably not going to be the same in every GRIB file, so the current solution might be very specific to our needs and therefore pretty fragile in case someone wants to use probtest outside ICON or even with different experiments. If we could find a way to distinguish between variables and these "dimensional fields" within the code, then we could simply fill the `short_names_excl` list using such criterion. I am not an expert with GRIB files, but I guess these variables (judging from their names) probably do not change with time, therefore they might not have a time dimension. Maybe we could leverage this and use it as a criterion to exclude them from probtest, assuming we are never interested in testing variables that are constant in time, which seems reasonable to me. Another uglier option could be to use whatever error those variables were causing to our advantage by handling it in a try-statement and adding the name of the variable to the `short_names_excl` list in case of error.

I think we could raise an issue for this, but maybe it is not super urgent. I guess it depends how much the list of excluded variables changes from file to file and how many different people will attempt to use probtest of GRIB files.

Sorry for the wall of text 😁

Reply:
Thanks for the wall of text :) 
I am not a grib expert, but I would assume it is true. We had a similar problem with the fof files, and there we just included files that have the specific horizontal dimensions, which excludes all the dimension variables.
But I think for now I would just put it all in a desperate issue.

Reply:
Yes, I agree! Will you open the issue or should I do it?